app/controllers/v1/application_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Superclass for all V1 controllers
+module V1
+  class ApplicationController < ::ApplicationController
```

Comment:
So far this can share all logic with the application controller we have for GraphQL

Reply:
This is acceptable IMO :+1: